imagepicker/src/main/java/com/esafirm/imagepicker/helper/ImagePickerUtils.java

```diff
@@ -79,4 +79,13 @@ public static boolean isGifFormat(Image image) {
         String extension = image.getPath().substring(image.getPath().lastIndexOf(".") + 1, image.getPath().length());
         return extension.equalsIgnoreCase("gif");
     }
+
+
+    public static boolean isVideoFormat(Image image) {
```

Comment:
I think checking the mime type for the path is a better approach rather than checking the extension to incomplete video extension. It could be something like [this](https://stackoverflow.com/a/30696106/2559752)